docs/commands.md

```diff
@@ -65,13 +65,15 @@ Commandfiles that don't ship inside Drupal modules are called 'global' commandfi
      - Filename: $PROJECT_ROOT/drush/Commands/ExampleCommands.php
      - Namespace: Drush\Commands
 1. Nested (e.g. Commandfile is part of a Composer package)
-    - Filename: $PROJECT_ROOT/drush/Commands/dev_modules/ExampleCommands.php
+    - Filename: $PROJECT_ROOT/drush/contrib/dev_modules/ExampleCommands.php
```

Comment:
Its a bit awkward to have `sites` alongside contrib/custom. Its more natural to me to have Commands alongside sites.